sdk/tables/data-tables/src/utils/connectionString.ts

```diff
@@ -20,6 +23,9 @@ export function getClientParamsFromConnectionString(
   connectionString: string,
   options?: TableServiceClientOptions
 ): ClientParamsFromConnectionString {
+  if (connectionString.indexOf("UseDevelopmentStorage=true") !== -1) {
```

Comment:
`UseDevelopmentStorage` isn't a case-sensitive attribute, in which case this will fail in scenarios such as `useDevelopmentStorage=true` and `UsEDeVeLoPmEntStorAge=truE`.

Reply:
Good catch, thanks!

Reply:
For reference here's documentation from ADO.NET where it mentions that connection string keys are case-insensitive while values may be case-sensitive

https://docs.microsoft.com/en-us/dotnet/framework/data/adonet/connection-strings#connection-string-syntax